build/helper/documentation_snippets.py

```diff
@@ -96,5 +96,97 @@
 { 'simulate': True, 'driver_setup': { 'Model': '<model number>',  'BoardType': '<type>' } }
 '''
 
+default_close_function_doc = '''
+Closes the driver session and cleans up.
+'''
+
+close_function_note = '''
+This function is not needed when using the session context manager
+'''
+
+def close_function_def_for_doc(functions):
+    # This is very specific to IVI. We look for a 'close' function and if we find it,
```

Comment:
NISE and niModInst have close. How is this specific to IVI?

Reply:
Changed to `# This is very specific to session based APIs ...`